shared/constants/chat2/meta.tsx

```diff
@@ -158,10 +158,10 @@ export const updateMeta = (
         if (shallowEqual(draft.nameParticipants, oldMeta.nameParticipants)) {
           draft.nameParticipants = oldMeta.nameParticipants
         }
-        if (shallowEqual(draft.rekeyers, oldMeta.rekeyers)) {
+        if (shallowEqual([...draft.rekeyers], [...oldMeta.rekeyers])) {
```

Comment:
immer 5 actually caused this to be caught. its invalid to pass a set or map to shallowEqual as its not a plain object